code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -139,6 +139,16 @@
 /mob/living/simple_animal/updatehealth()
 	..()
 	health = CLAMP(health, 0, maxHealth)
+	update_health_hud()
+
+/mob/living/simple_animal/update_health_hud()
+	if(!hud_used)
+		return
+	var/severity = 6 - (CLAMP(FLOOR((health / maxHealth) * 5, 1), 0, 5) + 1)
```

Comment:
```suggestion
	var/severity = 5 - CLAMP(FLOOR((health / maxHealth) * 5, 1), 0, 5)
```

Reply:
This is actually wrong, it would be 7 - x

Reply:
Nevermind, I'm stupid and did math wrong